### DISCUSSION TITLE
Jingle at start playlist

### DISCUSSION THREAD

**Discussion by masterjoy** [AzuraCast/AzuraCast#6011] [Q&A]
I did several tests by reading here but I still can't play a jingle at the beginning of a playlist.... can anyone tell me exactly how to set this up?
Thank you

**Comment by Mozitux** [comment]
Hello, currently, you must schedule a second playlist that contains your jingle at your playlist's start time.

**Reply by masterjoy** [reply]
I did but the music playlist starts first
